mobile/src/views/VerificationView.jsx

```diff
@@ -48,6 +48,21 @@ const VerifcationView = ({navigation}) => {
       })
       .then(data => {
         console.log(data);
+
+        fetch(API_ROUTE + 'api/students/' + user.id, {
+          method: 'get',
+          headers: {
+            'jwt-token': user['key'],
+            'x-api-key': API_KEY,
+            'Content-Type': 'multipart/form-data',
+          },
+        })
+          .then(result => {
+            return result.json()
+          })
+          .then(data => {
+            setUser(data)
+          })
```

Comment:
Maybe we want to add a `catch` block to handle errors.

Reply:
I added a catch block. I'm not too sure what else to add here though, if you have any suggestions